internal/dependency-track/setup.sh

```diff
@@ -0,0 +1,157 @@
+END
+
+sdk install java 8.0.212-amzn
+sdk install maven 3.6.1
+
+# -----------------------------
+# Change SQL Mode
+# -----------------------------
+
+sudo cat <<END >> /home/dependency-track/etc/mysql/my.cnf
+[mysqld]
+sql_mode="ANSI_QUOTES,STRICT_TRANS_TABLES,ONLY_FULL_GROUP_BY,ERROR_FOR_DIVISION_BY_ZERO,NO_AUTO_CREATE_USER,NO_ENGINE_SUBSTITUTION"
+END
+
+sudo service mysql restart
+
+# -----------------------------
+# Get MySQL connector
+# -----------------------------
+
+mkdir libs
+cd libs
+wget https://repo1.maven.org/maven2/mysql/mysql-connector-java/8.0.16/mysql-connector-java-8.0.16.jar
+cd -
+
+# --------------------------------------------------------
+# Get latest release of Dependency Track embedded war file
+# --------------------------------------------------------
+
+mkdir artifact
+cd artifact
+asset_type=dependency-track-embedded.war
+# Download only embedded war asset
+downloadUrl = $(curl -s https://api.github.com/repos/DependencyTrack/dependency-track/releases/latest | jq -r ".assets[] | select(.name | test(\"${asset_type}\")) | .browser_download_url")
+wget $downloadUrl
+cd -
+
+# ---------------------------------------------------
+# Get application.propertiesfile
+# ---------------------------------------------------
+
+mkdir config
+cd config
+wget https://raw.githubusercontent.com/wso2/security-tools/master/internal/dependency-track/config/application.properties
+cd -
+
+# ---------------------------------------------------
+# Configure Dependency Track database as MySQL
+# ---------------------------------------------------
+
+urlData='jdbc:mysql:\/\/localhost:3306\/dependency_track?autoReconnect=true\&useSSL=false'
+driver='com.mysql.cj.jdbc.Driver'
+driver_path='\/home\/dependency-track\/libs\/mysql-connector-java-8.0.16.jar'
+username='dependency-track'
+
+sed -i "s/alpine.database.url=/alpine.database.url=$urlData/" /home/dependency-track/config/application.properties
+sed -i "s/alpine.database.driver=/alpine.database.driver=$driver/" /home/dependency-track/config/application.properties
+sed -i "s/alpine.database.driver.path=/alpine.database.driver.path=$driver_path/" /home/dependency-track/config/application.properties
+sed -i "s/alpine.database.username=/alpine.database.username=$username/" /home/dependency-track/config/application.properties
+sed -i "s/alpine.database.password=/alpine.database.password=$dependency_track_password/" /home/dependency-track/config/application.properties
+
+# ---------------------------------------------------
+# Start Dependency Track server in nohub mode
+# ---------------------------------------------------
+
+cd artifact
+nohub java -Dalpine.application.properties=/home/dependency-track/config/application.properties -Xmx4G -jar dependency-track-embedded.war
+
+# ---------------------------------------------------
+# Alter privileges to Dependency Track Database
+# ---------------------------------------------------
+
+sudo mysql -u dependency-track -p
```

Comment:
IMO, we should do this after waiting for `DT` to run the DB creation scripts.

Can we ask the user to "press any key" after initialisation is done. Once user press a key only, this step should be executed.

Reply:
Done